packages/nimiq-vitepress-theme/src/layout/Layout.vue

```diff
@@ -42,7 +42,13 @@ const isMobileOrTablet = breakpoints.smaller('lg')
       </div>
       <main
         of-hidden
-        dark:bg-neutral-1100 min-h-screen flex-1 min-w-0 :class="{
+        dark:bg-neutral-1100 
+        min-h-screen 
+        flex-1 
+        min-w-0 
+        :class="{
+          'ml-[var(--nq-sidebar-width)]': showSidebar,
```

Comment:
small unnecessary thing.

With unocss this can be also written as `ml-[$nq-sidebar-width]`, which is easier :)